website/server/controllers/api-v3/groups.js

```diff
@@ -336,11 +336,7 @@ api.getGroups = {
     }
 
     if (req.query.search) {
-      filters.$or = [];
-      const searchWords = req.query.search.split(' ').join('|');
-      const searchQuery = { $regex: new RegExp(`${searchWords}`, 'i') };
-      filters.$or.push({name: searchQuery});
-      filters.$or.push({description: searchQuery});
+      filters.$text = { $search: req.query.search };
```

Comment:
Please add the $or queries back. The text index does not work for our requirements.